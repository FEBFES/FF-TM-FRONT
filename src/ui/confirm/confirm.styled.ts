import styled from 'styled-components';

/* //todo change to FLEX and PAddinWrapper ui Component */
export const SConfirmContainer = styled.div`
  padding: 40px 0px;
  display: flex;
  flex-direction: column;
  justify-content: flex-start;
  align-items: center;
`;

/* //todo change to FLEX and PAddinWrapper ui Component */
export const SButtonContainer = styled.div`
  display: flex;
  align-items: center;
  justify-content: center;
`;
