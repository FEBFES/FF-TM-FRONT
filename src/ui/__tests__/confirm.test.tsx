import React from 'react';
import { Confirm } from '../confirm/confirm';
import renderer from 'react-test-renderer';

it('SNAPSHOTS: UI - Confirm component - render', () => {
  const elem = renderer
    .create(
      <Confirm
        onConfirm={() => {}}
        show={true}
        setShow={() => {}}
        title={'????'}
      />
    )
    .toJSON();
  expect(elem).toMatchSnapshot();
});
