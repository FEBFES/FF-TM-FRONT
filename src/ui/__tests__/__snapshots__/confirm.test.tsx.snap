// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SNAPSHOTS: UI - Confirm component - render 1`] = `
<div
  className="sc-bcPKhP erxbYN"
  onClick={[Function]}
  show={true}
>
  <div
    className="sc-grXZZQ iBHSSL"
    onClick={[Function]}
  >
    <div
      className="sc-iBAaJG fQYHQA"
    >
      <h1
        className="sc-jSwlEQ jinZDO"
        size="h6"
      >
        ????
      </h1>
      <div
        className="sc-ftLKQv jQqpnh"
      >
        <button
          className="sc-eDnVMP kLJHxi"
          onClick={[Function]}
          variant="danger"
        />
        <button
          className="sc-eDnVMP hSYRyU"
          onClick={[Function]}
          variant="primary"
        />
      </div>
    </div>
  </div>
</div>
`;
